tests/test_vissharedmem_validation.py

```diff
@@ -0,0 +1,141 @@
+# === Start Python 2/3 compatibility
+from __future__ import absolute_import, division, print_function, unicode_literals
+from future.builtins import *  # noqa pylint: disable=W0401, W0614
+from future.builtins.disabled import *  # noqa pylint: disable=W0401, W0614
+
+# == End Python 2/3 compatibility
+
+import copy
+import logging
+import numpy as np
+import os
+import pytest
+import re
+import signal
+from subprocess import Popen
+import shutil
+import tempfile
+import threading
+from time import sleep
+
+from kotekan import runner, testing
+
+# use tempfile creation to get exclusive random strings
+useless_file = tempfile.NamedTemporaryFile()
+fname_buf = "calBuffer_" + os.path.split(useless_file.name)[-1]
+
+logging.basicConfig(level=logging.INFO)
+
+
+@pytest.fixture()
+def comet_broker():
+    broker_path = shutil.which("comet")
+    if not broker_path:
+        pytest.skip(
+            "Make sure PYTHONPATH is set to where the comet dataset broker is installed."
+        )
+
```

Comment:
Same here as in another comment. You should check to see if redis is actually available.